meson.build

```diff
@@ -0,0 +1,34 @@
+
+project('ashuffle', 'c', version: '1.0.3', default_options: ['c_std=c99'])
+
+add_global_arguments([
+	'-Wextra',
+	'-pedantic' ],
+	language: 'c'
+)
+
+sources = [
+	'src/args.c',
+	'src/ashuffle.c',
+	'src/getpass.c',
+	'src/list.c',
+	'src/rule.c',
+	'src/shuffle.c'
+]
+
+clientlib = dependency('libmpdclient')
+
+gdb = find_program('gdb', required: false)
+
+
+ashuffle = executable(
+	'ashuffle',
+	sources,
+	dependencies: clientlib,
+	install: true
+)
+
+
+run_target( 'gdb',
```

Comment:
This causes some weird build errors for me if `gdb` is not present. It would probably be best to just remove it. I only added it to have a quick shortcut for debugging :)

Errors:
```
/ashuffle # meson build
The Meson build system
Version: 0.48.2
Source dir: /ashuffle
Build dir: /ashuffle/build
Build type: native build
Project name: ashuffle
Project version: 1.0.3
Native C compiler: cc (gcc 8.2.0 "cc (Alpine 8.2.0) 8.2.0")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (1.5.4)
Dependency libmpdclient found: YES 2.16
Program gdb found: NO

meson.build:32:0: ERROR:  Tried to use non-existing executable 'nonexistingprogram'

A full log can be found at /ashuffle/build/meson-logs/meson-log.txt
```

Reply:
oh, damn, missed that... I had thought it would only complain if you'd try to use the target...
well, one still can do `ninja && gdb ./ashuffle` ^^